code/modules/food_and_drinks/food/candy.dm

```diff
@@ -134,14 +128,15 @@
 	trash = /obj/item/trash/candy
 	filling_color = "#7D5F46"
 	bitesize = 3
-	list_reagents = list("nutriment" = 2, "chocolate" = 5)
+	junkiness = 25
```

Comment:
I can understand your reasoning behind it, but it does work against the goal that I want the PR to have; for the chef to have more relevance and for the chef's food to always be more viable than the vending machine food.

That is to say, I want the distinction to be 100% black and white and clear cut. That is to say "chef good, vending machine bad".